test/tools/auth0/index.test.ts

```diff
@@ -8,10 +8,16 @@ const mockEmptyAssets = {} as Assets;
 describe('#Auth0 class', () => {
   describe('#resource exclusion', () => {
     it('should exclude handlers listed in AUTH0_EXCLUDED from Auth0 class', () => {
-      const auth0WithoutExclusions = new Auth0(mockEmptyClient, mockEmptyAssets, () => []);
+      const auth0WithoutExclusions = new Auth0(mockEmptyClient, mockEmptyAssets, (key) => {
+        const config = { AUTH0_EXCLUDED: [] };
+        return config[key];
+      });
 
       const AUTH0_EXCLUDED = ['rules', 'organizations', 'connections'];
-      const auth0WithExclusions = new Auth0(mockEmptyClient, mockEmptyAssets, () => AUTH0_EXCLUDED);
+      const auth0WithExclusions = new Auth0(mockEmptyClient, mockEmptyAssets, (key) => {
+        const config = { AUTH0_EXCLUDED };
+        return config[key];
+      });
```

Comment:
The configuration function injected into these tests were faulty and caused false-positives when the `AUTH0_INCLUDED_ONLY` property was added.